.github/workflows/ci.yml

```diff
@@ -41,7 +42,7 @@ jobs:
             otp: "24"
           - elixir: "1.10"
             otp: "23"
-          - elixir: "1.8"
```

Comment:
Why the drop of 1.8?

Reply:
I was using :persistent_term in `ex_sqlean` and 1.8 did not have it. I was hoping that it would be present in 1.9, but then I realised that it is an OTP lib, that was introduced in 21.2. So, I could actually make 1.8 the lowest supported Elixir version.

Reply:
I _think_ we can shift support for OTP to 21. People really shouldn't be on something that old.

Reply:
I actually agree. There are not many practical reasons to support older OTP for a new-ish (Elixir) SQLite library, considering how easy it is to upgrade your application to newer Erlang / Elixir versions.

Reply:
Let's bump the lowest to OTP 21 and try to support 1.8 elixir for a little while.